app/controllers/customers_controller.rb

```diff
@@ -2,4 +2,31 @@ class CustomersController < ApplicationController
   def index
     @customers = Customer.all
   end
-end
\ No newline at end of file
+
+  def show; end
+
+  def new
+    @customer = Customer.new
+  end
+
+  def create 
+    @customer = Customer.new(customer_params)
+    if @customer.save
+      redirect_to customers_path
+    else
+      render :new  
+    end
+  end
+
+  private
+
+  def customer_params
+    params.require(:customer).permit(:name, 
```

Comment:
```suggestion
    params.require(:customer).permit(
      :name,
      :phone,
      etc...
    )
```

Aqui pode indentar assim, acho que fica mais legal :smiley: